flake.nix

```diff
@@ -80,6 +80,14 @@
                                 RPC_PASS = lib.mkOption {
                                   type = types.str;
                                 };
+                                ADDRESS = lib.mkOption {
+                                  type = types.str;
```

Comment:
We might want to do some more thorough validation on the "Can be <IP>:<Port> or just <IP>" part here.